services/core-web/src/components/mine/NoticeOfDeparture/MineNoticeOfDepartureTable.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Button, Badge, Popconfirm } from "antd";
+import { EyeOutlined } from "@ant-design/icons";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import { formatDate } from "@common/utils/helpers";
+import * as Strings from "@common/constants/strings";
+import CustomPropTypes from "@/customPropTypes";
+import AuthorizationWrapper from "@/components/common/wrappers/AuthorizationWrapper";
+import * as Permission from "@/constants/permissions";
+import { RED_CLOCK, EDIT_OUTLINE_VIOLET, TRASHCAN } from "@/constants/assets";
+import * as router from "@/constants/routes";
+import CoreTable from "@/components/common/CoreTable";
+
+const NoticeOfDepartureType = {
```

Comment:
More and more I'm wishing we had a monorepo set up.  Every front end ticket is going to required varying levels of duplication of code that shouldn't be necessary.

Reply:
I think once we update the same in minespace, this will be part of common I guess?